### ISSUE TITLE
ubuntu on windows doesn't have `proc/sys/kernel/random/uuid` [Windows]

### ISSUE THREAD

**Issue by netpyoung** [Linuxbrew/brew#109]
If Homebrew was updated on Aug 10-11th 2016 and `brew update` always says `Already up-to-date.` you need to run: `cd "$(brew --repo)" && git fetch && git reset --hard origin/master && brew update`.
# Please follow the general troubleshooting steps first:
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

_You can erase any parts of this template not applicable to your Issue._
### Bug reports:

when i using linuxbrew on `linux on windows`, i got error message followed

`/home/pyoung/.linuxbrew/Library/Homebrew/utils/analytics.sh: line 41: /proc/sys/kernel/random/uuid: No such file or directory`

```
pyoung@DESKTOP-UNNNKLB:~/.linuxbrew$ uname -a
Linux DESKTOP-UNNNKLB 3.4.0+ #1 PREEMPT Thu Aug 1 17:06:05 CST 2013 x86_64 x86_64 x86_64 GNU/Linux
pyoung@DESKTOP-UNNNKLB:~/.linuxbrew$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.4 LTS
Release:        14.04
Codename:       trusty

pyoung@DESKTOP-UNNNKLB:~/.linuxbrew$ tree /proc/sys/kernel/
/proc/sys/kernel/
├── ngroups_max
├── osrelease
├── ostype
├── pid_max
└── randomize_va_space

0 directories, 5 files
```
### Propose a feature:

Instead of creating an issue here, please create a pull request with your change proposal in the [Homebrew Evolution](https://github.com/Homebrew/brew-evolution) repository using the [proposal template](https://github.com/Homebrew/brew-evolution/blob/master/proposal_template.md).

**Comment by netpyoung** [comment]
```
pyoung@DESKTOP-UNNNKLB:~/.linuxbrew$ brew doctor
/home/pyoung/.linuxbrew/Library/Homebrew/utils/analytics.sh: line 41: /proc/sys/kernel/random/uuid: No such file or directory
Your system is ready to brew.

pyoung@DESKTOP-UNNNKLB:~/.linuxbrew$ sudo chown -R $(whoami) $(brew --prefix)
sudo: unable to resolve host DESKTOP-UNNNKLB
```

**Comment by sjackman** [comment]
At the command line, can you please report the output of

``` sh
which -a uuidgen
```

This patch should hopefully work:

``` diff
diff --git a/Library/Homebrew/utils/analytics.sh b/Library/Homebrew/utils/analytics.sh
index 09eaf67..b685fb4 100644
--- a/Library/Homebrew/utils/analytics.sh
+++ b/Library/Homebrew/utils/analytics.sh
@@ -36,10 +36,10 @@ setup-analytics() {
   HOMEBREW_ANALYTICS_USER_UUID="$(git config --file="$git_config_file" --get homebrew.analyticsuuid)"
   if [[ -z "$HOMEBREW_ANALYTICS_USER_UUID" ]]
   then
-    if [[ -n "$HOMEBREW_LINUX" ]]
+    if [[ -r /proc/sys/kernel/random/uuid ]]
     then
       HOMEBREW_ANALYTICS_USER_UUID="$(tr a-f A-F </proc/sys/kernel/random/uuid)"
-    elif [[ -n "$HOMEBREW_OSX" ]]
+    elif [[ -x /usr/bin/uuidgen ]]
     then
       HOMEBREW_ANALYTICS_USER_UUID="$(/usr/bin/uuidgen)"
     else
```

**Comment by sjackman** [comment]
I've created a pull request: https://github.com/Linuxbrew/brew/pull/110

**Comment by sjackman** [comment]
Fixed by #110. Thanks for the bug report, @netpyoung!
